### DISCUSSION TITLE
Is it possible to change the contend or language of the invite E-Mails

### DISCUSSION THREAD

**Discussion by boospy** [TandoorRecipes/recipes#3347] [General]
Hello all, 

we are German speaker. So for some older friends it is difficult to understand English words, so it is there a way to  change the contend or language of the invite E-Mails

Thanks
greetings boospy

**Comment by vabene1111** [comment]
It should change automatically, at least I have some old emails that were send in german. 

What language is your user profile (the one creating the invite links)?
